frontend/src/views/ManagePermissionsView/index.vue

```diff
@@ -0,0 +1,329 @@
+            );
+            if (foundApp) {
+                setSelectedApp(foundApp);
+            }
+        }
+    },
+    { immediate: true } // Run immediately to catch initial load
+);
+
+// Available tab keys and their visibility conditions
+const tabs: ManagePermissionsTabTypes[] = [
+    {
+        key: AdminRoleAuthGroup.FamAdmin,
+        visible: computed(() => selectedApp.value?.id === 1),
+        icon: UserIcon as Component,
+    },
+    {
+        key: AdminRoleAuthGroup.AppAdmin,
+        visible: computed(
+            () =>
+                isSelectedAppAuthorized(
+                    "APP_ADMIN",
+                    selectedApp.value?.id,
+                    adminUserAccessQuery.data.value
+                ) ||
+                isSelectedAppAuthorized(
+                    "DELEGATED_ADMIN",
+                    selectedApp.value?.id,
+                    adminUserAccessQuery.data.value
+                )
+        ),
+        icon: UserIcon as Component,
+    },
+    {
+        key: AdminRoleAuthGroup.DelegatedAdmin,
+        visible: computed(
+            () =>
+                // DelegatedAdminFeatureFlag
+                !environment.isProdEnvironment() &&
+                isSelectedAppAuthorized(
+                    "APP_ADMIN",
+                    selectedApp.value?.id,
+                    adminUserAccessQuery.data.value
+                )
+        ),
+        icon: EnterpriseIcon as Component,
+    },
+];
+
+const notifications = ref<PermissionNotificationType[]>([
+    ...(appAdminSuccessData
+        ? generateAppSuccessNotifications(
+              "addUserPermission",
+              appAdminSuccessData
+          )
+        : []),
+    ...(delegatedAdminSuccessData
+        ? generateAppSuccessNotifications(
+              "addUserPermission",
+              delegatedAdminSuccessData
+          )
+        : []),
+    ...(appAdminErrorData
+        ? [generateAppErrorNotifications(appAdminErrorData)]
+        : []),
+    ...(delegatedAdminErrorData
+        ? [generateAppErrorNotifications(delegatedAdminErrorData)]
+        : []),
+    ...(FamPermissionSuccessData
+        ? [generateFamNotification(true, FamPermissionSuccessData)]
+        : []),
+    ...(FamPermissionErrorData
+        ? [generateFamNotification(false, FamPermissionErrorData)]
+        : []),
+]);
+
+const clearNotifications = () => {
+    queryClient.removeQueries({
+        queryKey: [AppAdminSuccessQuerykey],
+    });
+    queryClient.removeQueries({
+        queryKey: [AppAdminErrorQuerykey],
+    });
+    queryClient.removeQueries({
+        queryKey: [DelegatedAdminSuccessQueryKey],
+    });
+    queryClient.removeQueries({
+        queryKey: [DelegatedAdminErrorQueryKey],
+    });
+    queryClient.removeQueries({
+        queryKey: [FamPermissionSuccessQueryKey],
+    });
+    queryClient.removeQueries({
+        queryKey: [FamPermissionErrorQueryKey],
+    });
```

Comment:
we need to remove them since they are cached, and we fetches from these queries by query keys every time this View is being mounted, if we don't clear them when unmounted then the notifications will show up every time

Reply:
Probably I wasn't clear, I actually meant something else, like "pop" the item from the source (like stack for example) so there is no need to remove since you already call "pop".